terraform/shared/shared.tf.json.template.py

```diff
@@ -26,6 +26,12 @@
                 'lifecycle': {
                     'prevent_destroy': True
                 }
+            },
+            'config': {
+                'bucket': config.qualified_resource_name('config'),
```

Comment:
```suggestion
                'bucket': config.qualified_resource_name('config'),
```

As discussed during PL, we use a different naming scheme for buckets.